### PR TITLE
Preview Plan Removal

### PR DESCRIPTION
**Pull request by ralikio**
**Description**

Changes proposed in this pull request:

- Preview plan removal.

**Related issue(s)**

### CODE REVIEW

**Review by szwedm — APPROVED** [review]
/lgtm

**Review by szwedm — DISMISSED** [review]

**Review by klaudiagrz — DISMISSED** [review]

File: components/kyma-environment-broker/internal/runtimeversion/runtimeversion_test.go [hunk 1]
```diff
@@ -101,22 +100,20 @@ func Test_RuntimeVersionConfigurator_ForProvisioning_FromParameters(t *testing.T
 		require.Equal(t, 1, ver.MajorVersion)
 		require.Equal(t, internal.AccountMapping, ver.Origin)
 	})
-	t.Run("should return Kyma Version from ProvisioningParameters even when previewVersion provided", func(t *testing.T) {
+	t.Run("should return Kyma Version from ProvisioningParameters even when version provided", func(t *testing.T) {
+		// TODO: rename test above
```
Comment (wozniakjan) [inline components/kyma-environment-broker/internal/runtimeversion/runtimeversion_test.go:104]: nit: do we want to keep as todo or resolve as part of the PR?

**Review by wozniakjan — APPROVED** [review]
/lgtm
/approve
/hold

feel free to unhold if intending to keep the TODO

**Review by wozniakjan — APPROVED** [review]

**Review by majakurcius — APPROVED** [review]

**Pull request merged**